actionview/CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+*   Allow templates to define which locals they accept.
+
+    For example:
+
+    ```erb
+    <% accepts_locals(message:) -%>
```

Comment:
+1 for the comment. I think the Ruby method is confusing because though it's tecnically now valid syntax we aren't actually doing a method call (we are "defining" a method, though even that is a little abstracted to the user). Having this look like regular Ruby is likely to confuse users into thinking they could call the method dynamically.